Kim junho

xjvmdutl opened the pull request.

xjvmdutl committed:
feat  (Supplement) : 삭제 인수 테스트
- 영양제 삭제 인수테스트 작성
- 영양제 삭제 문서테스트 작성
- 영양제 삭제 컨트롤러 작성
- 영양제 삭제 서비스 작성

xjvmdutl committed:
Merge branch 'mandykr:main' into KimJunho

xjvmdutl committed:
feat  (Supplement) : 조회 인수 테스트 작성
- feature 작성
- 인수 테스트 작성

mandykr commented on src/main/java/com/example/nutrient/ui/SupplementController.java:
```diff
@@ -36,5 +39,10 @@ public ResponseEntity<SupplementUpdateResponse> update(
+        supplementService.delete(id);
+        return ResponseEntity.ok().build();
+    }
 }
```
[ PUT ]
특정 아이디에 해당하는 리소스를 대체한다는 의미로 URI를 설계하기 위해서는
Put API에서 Supplement id를 query param으로 넣어주는 것이 적당해 보입니다.

xjvmdutl committed:
feat  (Supplement) : Put , PathVariable을 통해 ID 받도록 수정

xjvmdutl commented:
update code

xjvmdutl committed:
feat  (Supplement) : Naver Open API 사용해서 최저가 검색
- application.properties : naverOpenApiURL, clientid, clientSecret 추가
- LowestPriceService : 네이버 최저가 API 호출
